brightwind/transform/transform.py

```diff
@@ -1556,18 +1556,17 @@ def apply_device_orientation_offset(data, meas_station_data_models, wdir_cols=[]
 def _check_vertical_profiler_properties_not_overlap(meas_station_data_models, df):
     """
     Checks if in vertical_profiler_properties there are any overlapping 
-    date ranges with non-zero device orientation values.
+    date ranges with device orientation values.
     
     Date ranges are considered as [from, to) where 'from' is inclusive and 'to' is exclusive.
-    Overlapping date ranges with non-zero device orientation values are not supported 
+    Overlapping date ranges with device orientation values are not supported 
     by apply_device_orientation_offset function and will raise an error.
     
     :param meas_station_data_models:    A simplified object to represent the IEA WRA Task 43 data model.
     :type meas_station_data_models:     brightwind.load.station.MeasurementStation
     :param df:                          DataFrame with the time series data
     :type df:                           pd.DataFrame
-    :raises ValueError:                 If overlapping date ranges with non-zero device orientation 
-                                        values are detected
+    :raises ValueError:                 If overlapping date ranges with device orientation values are detected
     :return:                            List of processed date ranges
```

Comment:
why this function is returning list of date ranges? isn't the goal only to return the error?